tests/drivers/mspi/trap_handler/testcase.yaml

```
@@ -0,0 +1,23 @@
+common:
+  tags:
+    - ci_tests_drivers_sdp
+    - drivers
+    - mspi
+  sysbuild: true
+  harness: console
+  harness_config:
+    type: one_line
+    regex:
+      - ">>> SDP APP FATAL ERROR"
+
+tests:
+  drivers.mspi.sdp_trap_handler:
+    platform_allow:
+      - nrf54l15dk/nrf54l15/cpuapp
+    integration_platforms:
+      - nrf54l15dk/nrf54l15/cpuapp
+    extra_args:
+      - EXTRA_DTC_OVERLAY_FILE="./boards/nrf54l15dk_nrf54l15_cpuapp_memory.overlay"
+      - sdp_CONFIG_ASSERT=y
+      - sdp_CONFIG_ASSERT_TEST=y
+      - sdp_EXTRA_DTC_OVERLAY_FILE="${ZEPHYR_NRF_MODULE_DIR}/tests/drivers/mspi/trap_handler/boards/nrf54l15dk_nrf54l15_cpuflpr_memory.overlay"
```

```suggestion
      - EXTRA_DTC_OVERLAY_FILE="./boards/nrf54l15dk_nrf54l15_cpuapp_memory.overlay"
      - sdp_CONFIG_ASSERT=y
      - sdp_CONFIG_ASSERT_TEST=y
      - SB_CONFIG_SDP_DEVELOPER_MODE=y
      - sdp_EXTRA_DTC_OVERLAY_FILE="${ZEPHYR_NRF_MODULE_DIR}/tests/drivers/mspi/trap_handler/boards/nrf54l15dk_nrf54l15_cpuflpr_memory.overlay"
```
Or, if the previous one does not work:
```suggestion
      - EXTRA_DTC_OVERLAY_FILE="./boards/nrf54l15dk_nrf54l15_cpuapp_memory.overlay"
      - sdp_CONFIG_ASSERT=y
      - sdp_CONFIG_ASSERT_TEST=y
      - sdp_CONFIG_SDP_DEVELOPER_MODE=y
      - sdp_EXTRA_DTC_OVERLAY_FILE="${ZEPHYR_NRF_MODULE_DIR}/tests/drivers/mspi/trap_handler/boards/nrf54l15dk_nrf54l15_cpuflpr_memory.overlay"
```